apis/pkg/v1alpha1/config.go

```diff
@@ -126,6 +130,17 @@ type ControllerConfigSpec struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
 
+// PodObjectMeta is metadata that is added to the Pods in a provider's
+// Deployment.
+type PodObjectMeta struct {
+	// Annotations is an unstructured key value map stored with a resource that may be
+	// set by external tools to store and retrieve arbitrary metadata. They are not
+	// queryable and should be preserved when modifying objects.
+	// More info: http://kubernetes.io/docs/user-guide/annotations
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Should we add `Labels`, too?

Reply:
I opted not to add `Labels` due to the semantics of `matchLabels` on the `Deployment` and `labels` on the pod spec template. We currently set a custom Crossplane label and I don't really want to enable adding additional ones or overwriting it until we have a use-case.

Reply:
Sounds good.